src/mapml-viewer.js

```diff
@@ -550,6 +550,32 @@ export class MapViewer extends HTMLElement {
         this.dispatchEvent(new CustomEvent('zoomend', {detail:
           {target: this}}));
       }, this);
+    this.addEventListener('fullscreenchange', function(event) {
+      if (document.fullscreenElement === null) {
+        // full-screen mode has been exited
+        this._map.contextMenu.setViewFullScreenInnerHTML('view');
+      } else {
+        this._map.contextMenu.setViewFullScreenInnerHTML('exit');
+      }
+    });
+    this.addEventListener('keydown', function(event) {
+      // Check if Ctrl+R is pressed and map is focused
+      if (event.ctrlKey && event.keyCode === 82 && document.activeElement.nodeName === "MAPML-VIEWER") {
+        // Prevent default browser behavior
+        event.preventDefault();
+        this.reload();
+      }
+      if (event.altKey && event.keyCode === 39 && document.activeElement.nodeName === "MAPML-VIEWER") {
+        // Prevent default browser behavior for Alt+Right-Arrow, if any
+        event.preventDefault();
+        this.forward();
+      }
+      if (event.altKey && event.keyCode === 37 && document.activeElement.nodeName === "MAPML-VIEWER") {
+        // Prevent default browser behavior for Alt+Left-Arrow (Back in browser history)
+        event.preventDefault();
+        this.back();
+      }
```

Comment:
it would be good to put all of this code inside the following if statement, so the condition isnt being checked 3 times:
```
if (document.activeElement.nodeName === "MAPML-VIEWER") {
      // code here
}
```
Similarly, instead of having 3 separate if  statements, you can use if/else if to save extra computation.

Reply:
Same thing for web-map